neurodebian/ANTs
# Debian based on upstream

Hi Yaroslav, 

My changes so far: 
- Add test data in debian/testdata 
- add soversion and version to libraries to remove lintian error: 
  _sharedobject-in-library-directory-missing-soname_
- move to debhelper 9 
- changed _libboost1.49-dev_ to version-less _libboost-dev_

The result is a package with the remaining lintian warnings and errors: 

```
lintian ants_2.0.0-1_amd64.deb    
```
- E: ants: _empty-manual-page_ usr/share/man/man1/GetMeshAndTopology.1.gz
- E: ants: _shell-script-fails-syntax-check_ usr/lib/ants/unbiased_pairwise_registration_with_aux_images.sh
  - See also https://github.com/stnava/ANTs/issues/70
  - For the moment we could just delete this script
- E: ants: _shell-script-fails-syntax-check_ usr/lib/ants/unbiased_pairwise_registration_with_aux_images.she
  - A left over from sed, I guess, why, I don't know. 
- W: ants: _executable-not-elf-or-script_ usr/share/modules/modulefiles/ants/ants.environment-modules
  This is something you add. Hence you should know if this needs to be corrected or overridden 
- N: 84 tags overridden (84 warnings),  Overrides are 
  - 7x errors in manpages. 
  - package-name-doesnt-match-sonames
  - non-dev-pkg-with-shlib-symlink

For the last one I could remove the *.so files from the install. Since there is no -dev package, the shared libraries with the version numbers should suffice. 

As for the test data. It is currently ~1MB uncompressed. I happily dig into packaging it into a separate tarball when I have more time, right now I don't see how git-buildpackage and a secondary source tarball may work together and would prefer to leave it in the debian\* part. Maybe when upstream is willing to add the data to their tarball the problem will resolve itself :) 

best 
Gert
